datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -96,6 +98,9 @@ def start(ctx, check, env, agent, dev, base, env_vars):
     else:
         agent_build = agent_ver.get(env_type, env_type)
 
+    if not isinstance(agent_ver, string_types) and use_jmx:
```

Comment:
?

Reply:
Its so that if a user specified `-a` we don't add `-jmx` to the end 🙂

Reply:
Have you tried it? 😝

Reply:
I have, works like a charm 😉

Reply:
⛓